src/main/java/sc/fiji/bdvpg/scijava/processors/BdvSourcePostprocessor.java

```diff
@@ -0,0 +1,55 @@
+package sc.fiji.bdvpg.scijava.processors;
+
+import bdv.viewer.Source;
+import org.scijava.module.Module;
+import org.scijava.module.process.AbstractPostprocessorPlugin;
+import org.scijava.module.process.PostprocessorPlugin;
+import org.scijava.plugin.Parameter;
+import org.scijava.plugin.Plugin;
+import sc.fiji.bdvpg.scijava.services.BdvSourceDisplayService;
+import sc.fiji.bdvpg.scijava.services.BdvSourceService;
+
+import java.util.function.Consumer;
+
+@Plugin(type = PostprocessorPlugin.class)
+public class BdvSourcePostprocessor extends AbstractPostprocessorPlugin {
+
+    @Parameter
+    BdvSourceService bss;
+
+    public static Consumer<String> log = (str) -> System.out.println(BdvSourcePostprocessor.class.getSimpleName()+":"+str);
```

Comment:
That's an interesting approach to logging. Why don't you use the SciJava `LogService` here?

Reply:
We are still not sure to which point we would like to use Scijava service, so we try to keep the dependencies minimal. This PR uses quite a lot the Scijava framework, because I find it convenient, but I'd like to keep open the option of not using it, if possible. Still an open discussion.

Reply:
Ah, alright. To be honest, I don't see any reason to *not* use SciJava wherever possible. You're writing `Command`s and `Plugin`s, so you depend on SciJava anyway. The `LogService` itself doesn't bring in any additional dependencies.

Reply:
The idea was to have bdv-related functionality in classes which don't use sci-java or imagej. We can then later merge functionality easier into bdv-core if desired... 😉